aws_health_to_slack/program.py

```diff
@@ -0,0 +1,115 @@
+"""Announce AWS health events in Slack, based on resource ownership in Google Sheet.
+
+Documentation:
+https://docs.aws.amazon.com/health/
+https://aws.amazon.com/blogs/mt/tag/aws-health-api/
+"""
+
+from datetime import UTC, datetime, timedelta
+import json
+import os
+import re
+
+import autokitteh
+from autokitteh.aws import boto3_client
+from autokitteh.google import google_sheets_client
+from autokitteh.slack import slack_client
+
+
+url = os.getenv("GOOGLE_SHEET_URL")
+
+
+def on_every_minute(_):
```

Comment:
i would change this to `on_schedule`, you might change the  manifest and then need to change this.